command/agent/command.go

```diff
@@ -135,6 +136,12 @@ func (c *Command) readConfig() *Config {
 			return nil
 		}
 
+		// The user asked us to load some config here but we didn't find any,
+		// so we'll complain but continue.
+		if current == nil || reflect.DeepEqual(current, &Config{}) {
```

Comment:
I think in the case where we couldn't find the configuration file, printing that message distinctly would be better.